x/auth/vesting/msg_server.go

```diff
@@ -118,19 +204,32 @@ func (s msgServer) CreatePeriodicVestingAccount(goCtx context.Context, msg *type
 		return nil, sdkerrors.Wrapf(sdkerrors.ErrUnauthorized, "%s is not allowed to receive funds", msg.ToAddress)
 	}
 
-	if acc := ak.GetAccount(ctx, to); acc != nil {
-		return nil, sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "account %s already exists", msg.ToAddress)
-	}
-
 	var totalCoins sdk.Coins
-
 	for _, period := range msg.VestingPeriods {
 		totalCoins = totalCoins.Add(period.Amount...)
 	}
+	totalCoins = totalCoins.Sort()
 
-	baseAccount := ak.NewAccountWithAddress(ctx, to)
+	acc := ak.GetAccount(ctx, to)
 
-	acc := types.NewPeriodicVestingAccount(baseAccount.(*authtypes.BaseAccount), totalCoins.Sort(), msg.StartTime, msg.VestingPeriods)
+	if acc != nil {
+		if !msg.Merge {
+			return nil, sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "account %s already exists", msg.ToAddress)
```

Comment:
Wording is up to you, but I think this is a good opportunity to help the user discover `--merge`.

```suggestion
			return nil, sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "account %s already exists; maybe specify --merge", msg.ToAddress)
```

Reply:
Done - restricted to just the case where it's a periodic vesting account.